ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockImporter.java

```diff
@@ -213,6 +217,10 @@ private String formatBlock(final SignedBeaconBlock block) {
     return LogFormatter.formatBlock(block.getSlot(), block.getRoot());
   }
 
+  private String getBlockContent(final SignedBeaconBlock block) {
+    return block.getMessage().sszSerialize().toHexString();
```

Comment:
We should log the whole signed block so just:
```suggestion
    return block.sszSerialize().toHexString();
```